CHANGELOG.md

```diff
@@ -4,7 +4,24 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
-## [v6.0.0](https://github.com/voxpupuli/puppet-openldap/tree/v6.0.0) (2022-11-21)
+## [v6.0.1](https://github.com/voxpupuli/puppet-openldap/tree/v6.0.1) (2023-02-15)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-openldap/compare/v6.0.0...v6.0.1)
+
+**Fixed bugs:**
+
+- Rename former herculesteam dependencies to reflect transfer to Vox [\#367](https://github.com/voxpupuli/puppet-openldap/pull/367) ([gcoxmoz](https://github.com/gcoxmoz))
+
+**Closed issues:**
+
+- openldap::server::dbindex is missing documentation [\#337](https://github.com/voxpupuli/puppet-openldap/issues/337)
```

Comment:
Is this a bug?

Reply:
Yeah, it is fixed in #367 visible above.  I do not think listing issues in the CHANGELOG make much sense, but it's the way we do this ATM.

Reply:
I thought we generally would mark the issue as a bug as well as the PR? If not, I'm fine with it as-is.

Reply:
FYI I edited my previous message where I mismatched the issue numbers :facepalm: .

I trend to keep the "bug" label for things broken in a way the behavior before and after is different.  This issue has a "docs" label (the fixing PR too), but GitHub ChangeLog Generator does not use this label.  I have no objections to mark them as "bugs" if it feels better categorized, but this still feels duplicated.

Maybe marking the issue "skip-changelog" would be acceptable?  I do not remember cases like this and don't know how we dealt with it previously…